motion_tracker/utils/image_cropping.py

```diff
@@ -0,0 +1,254 @@
+import cv2
+import os
+import numpy as np
+import pandas as pd
+from numpy.random import laplace
+
+class Coords(object):
+    '''Utility class to hold coords and automatically calculate height, width and center
+
+    x0, y0, x1 and y1 are all integers
+    '''
+
+    def __init__(self, x0, y0, x1, y1):
+        self.x0 = int(x0) # convert to int here. Otherwise shows up in many places
+        self.y0 = int(y0)
+        self.x1 = int(x1)
+        self.y1 = int(y1)
```

Comment:
Do I understand correctly that a lack of casting to `int` here really just causes problems for drawing bounding boxes during visualization? When I remove this and play with stuff, it doesn't cause any issues until I try to use your `show_stages_of_random_crop` function. So, I'm wondering if during training this is necessary or will be an issue? Removing it would save four calls to `int` here, and maybe more importantly (although I still don't see this having a big effect) is that we are throwing away some information by casting our bounding boxes to ints where previously they were floats. This change in the end probably won't matter, but it's maybe something to think about.